lib/main.dart

```diff
@@ -201,6 +216,41 @@ class _Item extends StatelessWidget {
   }
 }
 
+class _TabPage extends StatelessWidget {
```

Comment:
pulling this out is a good example of refactoring really simplifying code in a couple of places elsewhere. I'd recommend coding this up, or at least highlighting it in the next episode.